src/Sulu/Bundle/WebsiteBundle/Analytics/AnalyticsManager.php

```diff
@@ -73,17 +73,17 @@ public function __construct(
         $this->trashManager = $trashManager;
     }
 
-    public function findAll($webspaceKey)
+    public function findAll($webspaceKey): array
```

Comment:
BC break no return values should be added here:

```suggestion
    public function findAll($webspaceKey)
```